config-overrides.js

```diff
@@ -4,7 +4,7 @@ const rewireLess = require('react-app-rewire-less');
 module.exports = function override(config, env) {
   config = injectBabelPlugin(
     ['import', {libraryName: 'antd', style: true}],
-    config
+    config,
```

Comment:
A lot of times this prevent me from running npm, I wouldn't recommend adding a trailing comma here.

Reply:
Ah, so you're saying the current eslint configuration doesn't work with npm? Maybe we don't care that much as we're using `yarn` on this project, and it would make sense to just use the same configuration everywhere? (we also use it in delivery-dashboard).

Reply:
I can reproduce the error with yarn to actually:

```
rhubscher@rhubscher-ThinkPad-T460s:~/mozilla/react/moz-console$ yarn install
yarn install v1.3.2
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.1.2: The platform "linux" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "antd > rc-editor-mention@1.1.1" has unmet peer dependency "immutable@~3.7.4".
warning "antd > react-slick@0.15.4" has incorrect peer dependency "react@^0.14.0 || ^15.0.1".
warning "antd > react-slick@0.15.4" has incorrect peer dependency "react-dom@^0.14.0 || ^15.0.1".
warning "antd > react-slick > slick-carousel@1.8.1" has unmet peer dependency "jquery@>=1.8.0".
warning "photon-ant > less-loader@4.0.5" has unmet peer dependency "webpack@^2.0.0 || ^3.0.0".
[4/4] Building fresh packages...
Done in 8.53s.
rhubscher@rhubscher-ThinkPad-T460s:~/mozilla/react/moz-console$ yarn startt
yarn run v1.3.2
error Command "startt" not found. Did you mean "start"?
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
rhubscher@rhubscher-ThinkPad-T460s:~/mozilla/react/moz-console$ yarn start
yarn run v1.3.2
$ react-app-rewired start
/home/rhubscher/mozilla/react/moz-console/config-overrides.js:8
  );
  ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/rhubscher/mozilla/react/moz-console/node_modules/react-app-rewired/config-overrides.js:5:18)
    at Module._compile (module.js:570:32)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Instead I would remove it from the eslint checker.

Reply:
One way of fixing it is to use node 8 instead of node 6. Maybe we can detect when using node 6 and tell people to upgrade nicely?

Reply:
Fixed with be042a8